docs/source/fundamentals/benefits.md

```diff
@@ -0,0 +1,133 @@
+
+<h3 id="payload">Smaller payloads</h3>
+
+Since the response back from the server contains only the properties you specify in your query, GraphQL can significantly reduce payload size compared to a REST endpoint. Let's take a look at our dogs query from earlier in the article:
+
+```graphql
+const GET_DOGS = gql`
+  query {
+    dogs {
+      id
+      breed
+      image {
+        url
+      }
+      activities {
+        name
+      }
+    }
+  }
+`;
+```
+
+The response back from the server will be a list of dog objects with `id`, `breed`, `image`, and `activities` properties. It doesn't matter if the underlying REST endpoints we call in our resolvers return back objects with 100 properties! All of those extraneous properties will be filtered out before the response is sent back to the client.
+
+<h3 id="round-trip">Avoid round trips</h3>
+
+Since each GraphQL request returns only one response, switching to GraphQL can help you avoid costly round trips from the client to your server. With REST, each resource represents a round trip, which can quickly add up. If you're fetching items in a list, you'll have to complete a round trip for every resource multiplied by the number of items, causing slow load times especially on mobile devices.
+
+```bash
+GET /api/dogs/breeds
+GET /api/dogs/images
+GET /api/dogs/activities
+```
+
+With GraphQL, each query represents a single round trip from the client to server. If you'd like to reduce round trips even further, you can implement [query batching](https://www.apollographql.com/docs/engine/query-batching.html) to batch multiple queries into a single request. 
+
+<h2 id="dev-experience">Developer experience</h2>
+
+Implementing GraphQL in your organization via the Apollo platform can help you ship features faster due to its excellent developer experience. Our #1 goal is to simplify data management across the stack. Features that are normally difficult to execute, such as fullstack caching, data normalization, and optimistic UI suddenly become trivial thanks to Apollo Client, Apollo Server, and Apollo Engine. Let's learn how!
+
+<h3 id="data">Explore your API</h3>
+
+GraphQL's strongly typed query language allows developers to take advantage of incredible tooling for exploring GraphQL APIs. [GraphQL Playground](https://github.com/prismagraphql/graphql-playground) by Prisma is an excellent IDE featuring automatically generated docs for your schema and query execution with autocomplete. At a glance, you can see all the data available in your GraphQL API without diving into the backend code or knowing what source it came from.
+
+![GraphQL Playground](../assets/graphql-playground.png)
+
+Apollo Server 2.0 sets up GraphQL Playground out of the box, so you can start exploring your schema and making queries immediately. You also have access to GraphiQL within Apollo DevTools which is convenient for testing queries as you're working on front-end code with Apollo Client.
```

Comment:
I feel like Apollo DevTools deserves its own `h4` here?  Perhaps GraphQL Playground could also become one, but the `h3` for "Explore your API" could start by explaining how all this is possible because of introspection?

Reply:
Agreed! Just addressed all of these points and added a snazzy screenshot of the cache inspector for dev tools 👍